### ISSUE TITLE
Non-\n'd command execution leading to parsing failure

### ISSUE THREAD

**Issue by Headline** [Headline/discord-compiler-bot#42]
It's possible to trip up the command parser if requests are submitted in the following form

``````
;compile c++```cpp
int main() {
  return *(reinterpret_cast<int*>(0));
}```
``````

I don't have time to get into it right now, but it smells like it's coming from here
https://github.com/Headline/discord-compiler-bot/blob/8816dc786056aa8d515f0e1c02f09fd0388c6784/src/commands/utils/CompilationParser.js#L32

**Comment by ChristopherWMM** [comment]
@Headline I think it is actually just this snippet here:

https://github.com/Headline/discord-compiler-bot/blob/8816dc786056aa8d515f0e1c02f09fd0388c6784/src/commands/compile.js#L37-L43

In the case of the above input, `args[0]` will be the token `c++```cpp` and the if statement will fail as it isn't, itself, a recognized compiler or language. This ultimately stems from discord.js arguments being delimited by a space and that assumption being upheld in this implementation. The simplest patch would be to run the if check on the value in `arg[0]` up to the first \` is detected, if once exists, not `arg[0]` itself.

**Comment by Headline** [comment]
@ChristopherWMM great catch, thanks for looking into this. Do you have a patch incoming or should I swing?

**Comment by ChristopherWMM** [comment]
@Headline I can write one up and brush of the ol' js hat.
